SU-HSDO/suhumsci
# shs-56: fixes animation hero overlay

## Summary
* fixes animation hero overlay
* also fixes discovered bug of text not displaying with no-js

## Steps to Test
1. Go to the tug boat links for colorful and traditional below
2. On the Colorful site, go to this page: `/components/hero-images-main-region`
3. Verify that the text displays for the `Hero (Gradient)` (first) and the `Hero (Overlay)` (third) component
4. Go to the theme settings page `admin/appearance/settings/humsci_colorful`, check 'Use Animation Enhancements' and verify step 3 again
5. Turn off JavaScript and verify step 3 again
6. Change the theme to the Traditional theme and verify steps 3 and 4 again
